streaming_jupyter_integrations/sql_utils.py

```diff
@@ -54,6 +58,10 @@ def is_dql(sql: str) -> bool:
     return __first_token_is_keyword(sql, DQL_KEYWORDS)
 
 
+def shows_metadata(sql: str) -> bool:
```

Comment:
Nitpicking :smiley_cat: 
Because all other functions are starting with **is_...** then I was thinking if it is possible to add a function name similar here.
Perhaps **is_metadata_request** ?

Reply:
is_metadata_query?